src/clarity-angular/tooltips/_tooltips.clarity.scss

```diff
@@ -186,6 +186,7 @@ $clr-tooltip-adjusted-margin: $margin-value + $arrow-width * 2 !default;
     .tooltip {
         & > .clr-icon{
```

Comment:
Thanks @mathisscott for your suggestion. I have made the required changes and done a commit. Do I need to have a new Pull request too?